### DISCUSSION TITLE
Bitte zum Oled 2,42 1309 Display

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#1345] [General]
Hi,

da ich vom programmieren keine Ahnung habe möchte ich mal nachfragen ob es eventuell möglich ist das 1309 Display mit einzubauen.
Grundsätzlich funktioniert das Display und es wird auch alles angezeigt aber leider passt der Font irgendwie nicht weshalb manche Sachen etwas unscharf angezeigt werden.

Dann noch etwas anderes. Für das 1309 Display habe ich mir das Gehäuse https://www.printables.com/model/564101-open-dtu-gehause-242-zoll-display gedruckt und die Platine von https://shop.allianceapps.io/products/allianceapps-dtu-adapter-pcb-esp32-zu-nrf24l01. Das Display habe ich aber von Amazon.
Beim Ersteller des Gehäuses scheint die Platine mit dem Display ja zu funktionieren. Als ich heute gebastelt habe, habe ich es leider nicht geschafft das 2,42 Display zur Anzeige zu bewegen.
Nach sämtlichen versuchen habe ich dann mein 1306 Angeschlossen und auch hier blieb das Display dunkel. So dann habe ich wieder meine fliegende Verdrahtung aufgebaut und prompt funktionieren beide Displays.
Ich hab leider keinen Plan weshalb die Displays nicht mit der Platine funktionieren.
Die Platine Ansicht funktioniert aber da eine Verbindung zum HM-1500 aufgebaut wird und ich auch die Daten auf dem Pad angezeigt bekomme.
Hier noch ein Screenshot vom 2,42 Zoll Display mit aktueller Open Dtu Firmware.

![IMG_0024](https://github.com/tbnobody/OpenDTU/assets/17287119/20405b83-7aef-4d3e-9df4-98d40a19f78e)

**Comment by huste511** [comment]
Deine Links sind kaputt.
Welches Gehäuse oder Platine Du wo gekauft hast, sind für die Funktion vom Display etwas irrelevant.
Link zu dem genauem Display wäre hilfreicher.
Ob und wie Du das Protokoll vom Display ändern mußt und welche Pins zu nutzen sind, kann Dir am besten der Verkäufer sagen.

**Comment by jblond4711** [comment]
Hi,

mit dem Gehäuse gebe ich dir recht. Die Sache mit der Platine ist aber relevant da ja kein Display daran etwas anzeigt. Hat mich ja deshalb gewundert da mein 1.3 Zoll Display ja auch nichts anzeigt mit der Platine.
Hier der Link zum Display https://www.amazon.de/dp/B0B5BZLR2D?ref=ppx_yo2ov_dt_b_product_details&th=1
Auf dem Screenshot ist ja zu sehen das das Display funktioniert. Nur guck dir mal die Zeichen an die eigentlich Punkte sein sollen bei Datum und Uhrzeit.

**Comment by huste511** [comment]
Sorry, durch die vielen Baustellen hatte ich es vermischt und so verstanden, dass das Display nur bei dem anderem Typen funktioniert.
Wo genau soll den Dein Display unscharf sein? Ich sehe da nichts, außer der generellen Schriftart, die mir hier auch nicht so zusagt. ;-(
Pixel mal nachgezählt? Das sollte aber eigentlich stimmen.
Das ist aber auch ein Display aus China mit der üblichen Wartezeit. Hättest Du bei Ali für 1/3 bis die 1/2 bekommen.

Die Pluszeichen beim Datum habe ich auch. Bei mit hat die Uhrzeit aber 4 Pixel Quadrate.
Im Netz finden sich beim Datum aber auch noch Varianten mit Bindestrichen/Minuszeichen in Zeilenmitte als auch Schrägstriche.

Bin in die Displaygeschichte noch nicht tief eingestiegen, aber die Zeichen sollten durch einen font table bestimmt werden und dieser kommt mit openDTU mit. (?)

Wieso es einmal geht und einmal nicht, sollte sich mit einem Multimeter schnell durchpiepsen lassen.
btw: Bei machen Display ist plus/minus gerne mal vertauscht. Die Boards haben es ggf. auch mal so oder so.

Oft waren die Displays auch mit 5V befeuert. Die meisten halten es aus und die 3,3V vom ESP vertragen kaum Strom.
